lib/rubocop/cop/style/space_after_colon.rb

```diff
@@ -12,21 +12,33 @@ class SpaceAfterColon < Cop
 
         def on_pair(node)
           oper = node.loc.operator
-          return unless oper.is?(':') && followed_by_space?(oper)
 
-          add_offense(oper, oper)
+          return unless oper.is?(':')
+
+          add_offense_if_colon_not_followed_by_space(oper)
         end
 
         def on_if(node)
           return unless ternary_op?(node)
 
-          colon = node.loc.colon
-          return unless followed_by_space?(colon)
+          add_offense_if_colon_not_followed_by_space(node.loc.colon)
+        end
+
+        def on_kwoptarg(node)
+          # We have no direct reference to the colon source range following an
+          # optional keyword argument's name, so must construct one.
+          colon = node.loc.name.end.resize(1)
+
+          add_offense_if_colon_not_followed_by_space(colon)
+        end
+
+        private
 
-          add_offense(colon, colon)
+        def add_offense_if_colon_not_followed_by_space(colon)
```

Comment:
Fair enough. I thought the repeating of `colon` three times justified it (and it meant that I could've changed the name of `not_followed_by_space?` in one place rather than three)